botCommands/points.js

```diff
@@ -99,21 +95,20 @@ const awardPoints = {
           if (member && !member.roles.find(r => r.name==="club-40") && pointsUser.points > 39) {
             let pointsRole = guild.roles.find(r => r.name === "club-40")
             member.addRole(pointsRole)
-  
             let clubChannel = client.channels.get('707225752608964628')
+        
             if (clubChannel) {
               clubChannel.send(`HEYYY EVERYONE SAY HI TO ${user} the newest member of CLUB 40`)
             }
           }
-  
           channel.send(
             `${exclamation(pointsUser.points)} ${user} now has ${
               pointsUser.points
             } ${plural(pointsUser.points)}`
           );
         }
-      } catch (err) {}
-    });
+      } catch (err) {console.log(err)}
```

Comment:
We should probably remove this like the old code had.

Reply:
I might just throw an error here then? Otherwise we don’t have any idea what the error is, or really if it happens since we don’t do anything with this catch.

Reply:
I would keep it as the old code had, I'd hate for this to fill the logs up  until we can evaluate and refactor.